checks/http/cambium.rb

```diff
@@ -0,0 +1,41 @@
+module Intrigue
+    module Ident
+    module Check
+    class Cambium < Intrigue::Ident::Check::Base
+    
+      def generate_checks(url)
+        [
+          {
+            type: "fingerprint",
+            category: "hardware",
+            tags: ["Embedded", "Networking"],
+            vendor:"Cambium",
+            product: nil,
+            description: "page title",
+            match_type: :content_body,
```

Comment:
@lootfinder this can be content_title, see a list of all http matchers here - and note that match_types are specific to each protocol (content_banner is specific to protos with a banner as you clearly noticed): https://github.com/intrigueio/intrigue-ident#for-check-writers